modules/installation-rhv-customizing-install-config-yaml.adoc

```diff
@@ -15,7 +15,10 @@ Here, you use three Python scripts to override some of the installation program'
 
 * By default, the installation program sets the platform to `ovirt`. However, installing a cluster on user-provisioned infrastructure is more similar to installing a cluster on bare metal. Therefore, you delete the ovirt platform section from `install-config.yaml` and change the platform to `none`. Instead, you use `inventory.yml` to specify all of the required settings.
 
-NOTE: These snippets work with Python 3 and Python 2.
+[NOTE]
```

Comment:
Maybe use Python 3.x and Python 2.x to imply all the versions work as well?

Reply:
I'd probably rather not change that, since it might change the technical meaning (and I don't plan on getting QE review on this PR).